shell/browser/browser.h

```diff
@@ -120,6 +134,13 @@ class Browser : public WindowListObserver {
     base::string16 path;
     std::vector<base::string16> args;
 
+#if defined(OS_WIN)
+    bool enabled = true;
+    bool executable_will_launch_at_login = false;
+    std::vector<LaunchItem> launch_items;
+    base::string16 name = base::string16();
```

Comment:
Are these used?

Reply:
enabled can be passed in setLoginItemSettings();
as can name

executable_will_launch_at_login and launch_items are part of the loginItemSettings object that is returned from getLoginItemSettings();